client/component/people.js

```diff
@@ -19,14 +20,34 @@ export function ProfilePhoto({userId, type='large', photo=null, faint=false, che
     } else {
         const face = persona?.face;
         if (face || photo || persona?.photoUrl) {
-            return <FaceImage face={face} photoUrl={photo ?? persona?.photoUrl} type={type} 
-                border={border} faint={faint} check={check} />
+            return <> 
+                <FaceImage face={face} photoUrl={photo ?? persona?.photoUrl} type={type} border={border} faint={faint} check={check} />
+                {renderLayers(type, persona)}
+            </>
         } else if (persona?.hue && persona?.name) {
             return <LetterFace name={persona.name} hue={persona.hue} type={type} />
         } else {
-            return <AnonymousFace faint={faint} type={type} border={border} />    
+            return <AnonymousFace faint={faint} type={type} border={border} />
         }
     }
+
+}
+
+function renderLayers(type, persona) { 
+
+    const { layersByline } = useConfig();
```

Comment:
can we call this profilePhotoLayers?

Normal convention is that the name of a config slot starts with the context in which it appears.

Plus, it seems that the layers apply to any profile photo, not just those that appear in bylines, right?

Reply:
yes you're right , naming updated